keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -135,7 +135,7 @@ Below is a table describing which stats come from what source now, as well a pro
 |                              |N/A                   |container_memory_failcnt                        |N/A                             |cAdvisor              |CRI or N/A                 |  memory.failcnt                N/A
 |                              |N/A                   |container_memory_failures_total                 |N/A                             |cAdvisor              |CRI or N/A                 | (memory.stat) pg_fault && pg_maj_fault |
 |                              |N/A                   |container_memory_mapped_file                    |N/A                             |cAdvisor              |CRI or N/A                 | (memory.stat) mapped_file  |  (memory.stat) file_mapped
-|                              |N/A                   |container_memory_max_usage_bytes                |N/A                             |cAdvisor              |CRI or N/A                 | memory.max_usage_in_bytes  |  memory.max
+|                              |N/A                   |container_memory_max_usage_bytes                |N/A                             |cAdvisor              |CRI or N/A                 | memory.max_usage_in_bytes  |  memory.peak
```

Comment:
should it be conditioned depending on a kernel version than?

Reply:
[v5.19](https://github.com/torvalds/linux/releases/tag/v5.19) : https://github.com/torvalds/linux/commit/8e20d4b332660a32e842e20c34cfc3b3456bc6dc

Reply:
No kernel version are mentioned in the KEP, so I added the link and kernel version in commit message.

Reply:
My ask is to mention the minimum kernel version in this KEP readme.

Reply:
Updated.